config.js

```diff
@@ -80,12 +80,14 @@ config.cookie = {
 /*!
  * The servers related configuration
  *
+ * @property  {String}      serverHostname          The hostname on which the web servers should be listening. See https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback
  * @property  {String}      adminHostname           The hostname on which the admin UI will be made available
  * @property  {Number}      adminPort               The port on which the admin API endpoints will be made available
  * @property  {Number}      appsPort                The port on which the regular application API endpoints will be made available
  * @property  {String}      shibbolethSPHost        The hostname on which the Shibboleth Service Provider software will be made available
  */
 config.servers = {
+    'webServerHostname': '0.0.0.0',
```

Comment:
Thanks, it seems from those docs that "listen" is the directive normally used. I would be fine with just "listenAddress", express is just an implementation detail.